src/python/pants/option/global_options.py

```diff
@@ -273,6 +273,16 @@ def register_bootstrap_options(cls, register):
       help='Create a new pantsd server, and use it, and shut it down immediately after. '
            'If pantsd is already running, it will shut it down and spawn a new instance (Beta)')
 
+    # This determines how long a request to the Pailgun Server could block for before failing.
+    #
+    # We really don't want this option to invalidate the daemon, because different clients might have
+    # different needs. For instance, an IDE might have a very long timeout because it only wants to refresh
+    # a project in the background, while a user might want a shorter timeout for interactivity.
+    register('--pantsd-run-start-timeout', advanced=True, type=float, default=60.0, daemon=False,
```

Comment:
This is a bit difficult to understand what it means, I think because it has so many nouns all in a row.

I recommend either `--pantsd-start-timeout`, `--pantsd-start-run-timeout`, or `--pantsd-start-command-timeout`.

Reply:
👍